src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala

```diff
@@ -117,9 +117,10 @@ class EasyFedora2vaultApp(configuration: Configuration) extends DebugEnhancedLog
       _ <- managedMetadataStream(foXml, "DATASET_LICENSE", bag, "depositor-info/depositor-agreement") // TODO EASY-2697: older versions
         .getOrElse(Success(()))
       fedoraIDs <- fedoraProvider.getSubordinates(datasetId)
-      _ <- fedoraIDs.toStream
+      fileMetadata <- fedoraIDs.toStream
         .withFilter(_.startsWith("easy-file:"))
-        .map(addPayloadFileTo(bag)).failFastOr(Success(()))
+        .map(addPayloadFileTo(bag)).collectResults
```

Comment:
why change from failfast to failslow?

Reply:
Previously I did not need results. Added a note to https://drivenbydata.atlassian.net/browse/EASY-2722